mantle/kola/harness.go

```diff
@@ -447,29 +449,39 @@ func ParseDenyListYaml(pltfrm string) error {
 			continue
 		}
 
+		// Process "special" patterns which aren't test names, but influence overall behavior
+		if obj.Pattern == SkipConsoleWarningsTag {
+			SkipConsoleWarnings = true
+			continue
+		}
+
 		if obj.SnoozeDate != "" {
 			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
 			if err != nil {
 				return err
-			} else if today.After(snoozeDate) {
-				continue
 			}
-
-			fmt.Printf("🕒 Snoozing kola test pattern \"%s\" until %s:\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
+			if today.After(snoozeDate) {
+				fmt.Printf("⏰  Snooze for kola test pattern \"%s\" expired on: %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

Comment:
```suggestion
				fmt.Printf("⏰ Snooze for kola test pattern \"%s\" expired on %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
```

The spacing for some emojis is weird. Just ran this locally and it looks better if we remove one of these spaces.

Also remove the `:`.